elisp/intero.el

```diff
@@ -350,9 +350,10 @@ Returns nil when unable to find definition."
           (xref-push-marker-stack)
         (with-no-warnings
           (ring-insert find-tag-marker-ring (point-marker))))
-      (let ((file (match-string 1 result))
-            (line (string-to-number (match-string 2 result)))
-            (col (string-to-number (match-string 3 result))))
+      (let* ((returned-file (match-string 1 result))
+	     (line (string-to-number (match-string 2 result)))
+	     (col (string-to-number (match-string 3 result)))
+	     (file (intero-extend-path-by-buffer-host returned-file)))
```

Comment:
So this maps from `/foo/bar.hs` to `/ssh:blah.com/foo/bar/.hs`, right? Because the `intero` piped process returns a local path, whereas we want a remote-path, in order to open the right TRAMP buffer.